devicetypes/rooms-beautiful/curtain.src/curtain.groovy

```diff
@@ -0,0 +1,260 @@
+/**
+ *
+ *	Copyright 2019 SmartThings
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ */
+
+import groovy.json.JsonOutput
+import physicalgraph.zigbee.zcl.DataType
+
+metadata {
+	definition(name: "Curtain", namespace: "Rooms Beautiful", author: "Alex Feng", ocfDeviceType: "oic.d.blind", mnmn: "SmartThings", vid: "generic-shade-4") {
```

Comment:
Please use `Rooms Beautiful Curtain` as the DTH name. You'll also need to change the file name from `devicetypes/rooms-beautiful/curtain.src/curtain.groovy ` to `devicetypes/rooms-beautiful/rooms-beautiful-curtain.src/rooms-beautiful-curtain.groovy `

Reply:
@dkirker I've now changed this in the DTH and committed it.

Reply:
@dkirker In regards to the indenting, on the Smartthings IDE, the indenting looks good, but on GitHub, it looks messed up a bit. I'm not sure how to fix that.

Reply:
@devropes I should be able to assist with that. Unfortunately, the IDE mixes spaces and tabs.

Reply:
@dkirker I use graph.api.smartthings.com as the IDE. Is this there a newer IDE than this?
And I can fix the tabbing directly on the github website for now.
Also, should I remove curtain.src/curtain.groovy from the repo?

Reply:
@dkirker Actually I will probably need assistance with the indenting. It doesn't seem like editing in GitHub helps.

Reply:
@devropes I'll take care of the indenting. I can also deal with the duplicate files at the same time.